### DISCUSSION TITLE
Schaltzeiten passen nicht

### DISCUSSION THREAD

**Discussion by KingLuy** [camueller/SmartApplianceEnabler#493] [Q&A] [answered]
Hallo zusammen,

erstmal Danke für das coole Projekt. Hat erstmal Spaß gemacht alles so einigermaßen zum laufen zu bekommen.

Jetzt zu meiner Frage. Ich habe ein Gerät (Shelly Plug S) angelegt und dazu einen Schaltzeiten Plan erstellt.
![image](https://user-images.githubusercontent.com/3822777/223225973-e9af3ac9-96fb-4da2-9fc3-9e6194fc939f.png)
Da möchte ich zwischen 10 und 15 Uhr mindestens 1-2,5h meinen Akku vom Wohnwagen laden. 
Die Ampel sagt:
![image](https://user-images.githubusercontent.com/3822777/223226248-ab308026-861a-42bc-a3a5-ef752fb4d1a3.png)
Start frühestens in 13h 16min
Uhrzeit ist jetzt 21:44
Wenn ich hochrechner startet er frühstens um 11 Uhr oder hab ich wo nen Knoten im Hirn?!

Grüße
King Luy

**Comment by camueller** [comment]
Du schreibst, was die Urhzeit "jetzt" ist, aber ist sie das auch auf dem Sysetm, auf dem der SAE läuft?
Ist die Zeitzone dort korrekt eingestellt?

**Reply by KingLuy** [reply]
```
sae@raspberrypi:~ $ date
Mon  6 Mar 21:57:45 CET 2023
sae@raspberrypi:~ $ date +"%Z %z"
CET +0100
```
Passt, das hatte ich auch erst im Verdacht.

**Reply by KingLuy** [reply]
Ich sehe aber gerade, im Log file fehlt auch eine Stunde, da ist erst jetzt 21 Uhr

**Comment by KingLuy** [comment, marked as answer]
Ich hab jetzt nochmal die Zeitzone über raspi-config gesetzt und nochmal neu gestartet. Jetzt scheint es zu passen. Ich hab schon an mir gezweifelt. Scheint ein interner Raspberry Pi Fehler gewesen zu sein. Trotzdem Danke!
